### PR TITLE
update building image pipeline

### PR DESCRIPTION
**Pull request by smusali**
There has been a bug in building and releasing Docker image - this `Pull Request` is intended to fix

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,6 +7,11 @@ executors:
       MAJOR_VERSION: 1.
     docker:
       - image: circleci/buildpack-deps:stretch
+tagged_build_filters: &tagged_build_filters
```
Comment (RYadav95) [inline .circleci/config.yml]: This is pretty cool. Is this `type` defined in CircleCI? Wasn't able to find any documentation.

Comment (smusali) [reply .circleci/config.yml]: @RYadav95, I have already tested that pattern on some others - added with @mindjiver's suggestion

Comment (RYadav95) [reply .circleci/config.yml]: Yeah!! I was just wondering where you guys got this idea from. If I can read a little more about this for my own knowledge

Comment (mindjiver) [reply .circleci/config.yml]: I'll hook you up with some docs tomorrow.

**Pull request merged**